phenotrex/io/flat.py

```diff
@@ -59,35 +59,42 @@ def load_genotype_file(input_file: str) -> List[GenotypeRecord]:
     :return: List[GenotypeRecord] of records in the genotype file
     """
     with open(input_file) as genotype_file:
+        metadata = dict()
+        genotype_lines = []
         genotype_records = []
         for line in genotype_file:
+            if line.startswith('#'):
```

Comment:
Hm, what about lines starting with `[space]#` or the like? Do we want to allow that, too?
Perhaps:
```suggestion
            if line.strip().startswith('#'):
```

Reply:
It's a matter if taste I guess but I think there's no harm in allowing whitespace left of the header. Can do